models/DataObject/Localizedfield/Dao.php

```diff
@@ -849,7 +849,7 @@ public function createUpdateTable($params = [])
                                 $this->addModifyColumn($table, $key . '__' . $fkey, $fvalue, '', 'NULL');
                                 $protectedColumns[] = $key . '__' . $fkey;
                             }
-                        } elseif ($value->getColumnType()) {
```

Comment:
return value can be null

Reply:
See https://github.com/pimcore/pimcore/blob/9c2c366e7b96b5994828bb8c33d5561737cb7383/models/DataObject/Localizedfield/Dao.php#L843